staging/src/k8s.io/component-base/metrics/testutil/metrics_test.go

```diff
@@ -265,3 +265,91 @@ func TestHistogramValidate(t *testing.T) {
 		}
 	}
 }
+
+func TestLabelsMatch(t *testing.T) {
+	cases := []struct {
+		name          string
+		metric        *dto.Metric
+		labelFilter   map[string]string
+		expectedMatch bool
+	}{
+		{"metric's labels and labelFilter have the same labels and values", &dto.Metric{
```

Comment:
Sorry for reply late.
Would you mind replace `metric's` with `metric`? That'll be ugly if subtest's name contains a specific symbol.

Same as below.

And, why not set fields name here?  like,
```
{
    name: "...",
    metric: ...
    ....
}
```

Reply:
ok . I have replace `metric's` with `metric`. 
And for fields, I have updated it though i think both are ok.
Thanks for your reply.